### PR TITLE
SD-2321 As a user I want to select multiple items and perform SPIKE or UNSPIKE actions

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

File: client/spec/helpers/content.js [hunk 1]
```diff
@@ -23,6 +23,10 @@ function Content() {
         });
     };
 
+    this.getItems = function(item) {
```
Comment (ioanpocol) [inline client/spec/helpers/content.js]: Why item is needed?

File: client/spec/helpers/content.js [hunk 2]
```diff
@@ -23,6 +23,10 @@ function Content() {
...
+        return element.all(by.repeater('items._items'));
+    };
+
     this.getItem = function(item) {
         return element.all(by.repeater('items._items')).get(item);
```
Comment (ioanpocol) [inline client/spec/helpers/content.js]: Can be refactored to use getItems.

**Comment by ioanpocol** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Comment by plamut** [comment]
This should be refactored, it suffers from the same problem that Ioan fixed in #699 (the `sdMultiActionBar` directive might accidentally overwrite things in the parent element's scope). It would be best to bind these spike/unspike methods to the controller as well, just like in the linked PR.

**Comment by ioanpocol** [comment]
Already done on last update of #699

**Comment by plamut** [comment]
Ah, good then!

**Pull request merged**